gum/backend-x86/gumstalker-x86.c

```diff
@@ -4723,80 +4961,73 @@ gum_exec_block_write_sysenter_continuation_code (GumExecBlock * block,
                                                  GumGeneratorContext * gc,
                                                  gpointer saved_ret_addr)
 {
+  GumStalker * stalker = block->ctx->stalker;
+  const gint trust_threshold = stalker->trust_threshold;
   GumX86Writer * cw = gc->code_writer;
-  gconstpointer resolve_dynamically_label = cw->code;
+  GumX86Writer * cws = gc->slow_writer;
+  gpointer * ic_match;
 
-  gum_x86_writer_put_mov_reg_near_ptr (cw, GUM_REG_EDX,
+  if (trust_threshold >= 0)
+  {
+    if ((block->ctx->sink_mask & GUM_RET) != 0)
+    {
+      gum_exec_block_write_ret_event_code (block, gc, GUM_CODE_UNINTERRUPTIBLE);
+    }
+
+    gum_exec_block_open_prolog (block, GUM_PROLOG_IC, gc, gc->code_writer);
+
+    /* But first, check if we've been asked to unfollow,
+    * in which case we'll enter the Stalker so the unfollow can
+    * be completed... */
```

Comment:
Alignment is off. This could probably also be reflowed to use more of the width.